src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -1734,7 +1733,27 @@ public ResponseEntity<Void> invalidateShortCircuitStatus(@Parameter(description
         @ApiResponse(responseCode = "200", description = "The informations on all known servers"),
         @ApiResponse(responseCode = "207", description = "Partial result because some servers haven't responded or threw an error"),
         @ApiResponse(responseCode = "424", description = "All requests have failed, no informations retrieved")})
-    public ResponseEntity<Map<String, ?>> getServersInformations() { //Map<String, Info> from springboot-actuator
-        return ResponseEntity.ok().body(remoteServicesInspector.getServicesInfo());
+    public ResponseEntity<Map<String, JsonNode>> getServersInformations(
+            @Parameter(description = "filter services returned for specific front") @RequestParam final Optional<FrontService> view
+    ) { //Map<String, Info> from springboot-actuator
+        try {
+            return ResponseEntity.ok(remoteServicesInspector.getServicesInfo(view.orElse(null)));
+        } catch (final PartialResultException e) {
+            return ResponseEntity.status(HttpStatus.MULTI_STATUS).body((Map<String, JsonNode>) e.getResult());
+        }
+    }
+
+    @GetMapping(value = "/servers/about")
+    @Operation(summary = "Get the aggregated about informations from all backend servers")
```

Comment:
Not all because there is a "filter" :)

Reply:
Please update both endpoints

Reply:
but its specified in the description of the parameter?
![image](https://github.com/gridsuite/study-server/assets/135599584/82d3c287-24dc-4105-ada0-31cb5b89a48b)

Reply:
Yeah you should change both description imo